app/serializers/program_serializer.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class ProgramSerializer < ActiveModel::Serializer
+  attribute :facility_code
+  attribute :institution_name
```

Comment:
The two above attributes should probably be removed.  It's redundant data that is available on the `institutions` object.

Reply:
That's a 10-4. Changes are in.